modules/s3db/req.py

```diff
@@ -2122,13 +2122,15 @@ def model(self):
 
         list_fields = ["id",
                        "skill_id",
-                       # @ToDo: Activate based on a deployment_setting
-                       #"task",
                        "quantity",
                        "quantity_transit",
                        "quantity_fulfil",
                        "comments",
                        ]
+
+        if settings.get_req_task_list_field():
+            list_fields.insert(2, "task")
```

Comment:
There isn't a field called task.  Looks like this ToDo goes with the comment in the req_req_skills table definition that says to make task a component.  One presumes the purpose is to allow multiple tasks, else it could just be a foreign key reference to the project_task table (?), or a text string if they don't mean a project_task.

So, what we need to find out is, do they want "task" to be a reference to a project_task?

Then you'd define it as a component, probably by adding a link table between req_req_skill and project_task.  The line in list_fields wouldn't be just bare "task", but would be the appropriate form for link tables.  The project module would need to be loaded if this gets used.  Might need to see how that's handled in other cases -- does it have to go in the template config.py? or is there another way to inlcude it automagically?